iceberg/iceberg-handler/src/test/results/positive/vectorized_iceberg_merge_mixed.q.out

```diff
@@ -533,7 +533,7 @@ STAGE PLANS:
                 TableScan
                   alias: store_sales
                   filterExpr: ((ss_sold_date_sk = 2451181) and ss_item_sk is not null and ss_customer_sk is not null) (type: boolean)
-                  Statistics: Num rows: 2 Data size: 35#### Basic stats: COMPLETE Column stats: COMPLETE
+                  Statistics: Num rows: 2 Data size: 11#### Basic stats: COMPLETE Column stats: COMPLETE
```

Comment:
why did it affect the stats?

Reply:
Its not stats. It actually reduced the datasize which I see as a clear advantage of this PR.

Reply:
but i don't see that this optimization kicks in here

Reply:
A delete query is present here - 
https://github.com/apache/hive/blob/master/iceberg/iceberg-handler/src/test/queries/positive/vectorized_iceberg_merge_mixed.q#L47